CMakeLists.txt

```diff
@@ -102,11 +102,7 @@ endif()
 message(STATUS "GIT_SHA1 ${GIT_SHA1}")
 
 set(SYSCONFDIR ${SYSCONF_INSTALL_DIR})
-set(DATADIR ${DATA_INSTALL_DIR})
-if(WIN32)
-set(DATADIR "share")
-endif(WIN32)
-set(SHAREDIR ${DATADIR})
+set(SHAREDIR ${CMAKE_INSTALL_DATADIR})
```

Comment:
I think `SHAREDIR` can now be deleted.

Reply:
hm, actually it is still needed for https://github.com/nextcloud/desktop/blob/master/config.h.in#L37. For reasons I don't really understand it doesn't work when I use CMAKE_INSTALL_DATADIR instead

Reply:
I think that's because `#cmakedefine` expects the name of the value to be equal to name of the variable. If you use `#define` it works fine. Bu I have not found yet something in the documentation that explains that exact.

Reply:
might well be true but documentation is lacking
let's not block this due to that